code/game/turfs/simulated/lava.dm

```diff
@@ -168,6 +168,8 @@
 	smooth = SMOOTH_MORE | SMOOTH_BORDER
 	canSmoothWith = list(/turf/open/lava/smooth)
 
+
+
```

Comment:
Random whitespace addition?

Reply:
ah that was what was left over from my hacky fix of the nest atmos before i used fastdmm.
forgot to get rid of the extra lines after i got rid of the commented out initial atmos.

thanks for looking over this ill make those changes now